pkg/api/validation/validation.go

```diff
@@ -58,6 +58,8 @@ var DNS1123LabelErrorMsg string = fmt.Sprintf(`must be a DNS label (at most %d c
 var DNS952LabelErrorMsg string = fmt.Sprintf(`must be a DNS 952 label (at most %d characters, matching regex %s): e.g. "my-name"`, validation.DNS952LabelMaxLength, validation.DNS952LabelFmt)
 var pdPartitionErrorMsg string = IntervalErrorMsg(0, 255)
 var PortRangeErrorMsg string = IntervalErrorMsg(0, 65536)
+var GroupIdRangeErrorMsg string = IntervalErrorMsg(-1, 2147483648)
```

Comment:
GIDs and UIDs are *usually* unsigned value, with some small number of negative values having meaning.

Maybe the limits should be `(math.MaxUint32 - 16)` or if we want to be not-exactly-compatible `math.MaxInt32`